python/pythonmonkey/builtin_modules/base64.py

```diff
@@ -8,7 +8,10 @@
 
 
 def atob(b64):
-  return str(base64.standard_b64decode(b64), 'latin1')
+  # Workaround for pythons 'Incorrect padding' error when base64 decoding, see: 
+  # https://stackoverflow.com/questions/2941995/python-ignore-incorrect-padding-error-when-base64-decoding
+  append = b"==" if isinstance(b64, bytes) else "=="
+  return str(base64.standard_b64decode(b64 + append), 'latin1')
```

Comment:
Would it be better to add a comment here saying that excess padding will be ignored?

Reply:
Alternatively, for readability, it could add the exact amount of padding as needed.

```py
padding = '=' * (4 - (len(data) & 3))
```

[source](https://packaging.python.org/en/latest/specifications/binary-distribution-format/#appendix)

Reply:
> Alternatively, for readability, it could add the exact amount of padding as needed.
> 
> ```python
> padding = '=' * (4 - (len(data) & 3))
> ```
> 
> [source](https://packaging.python.org/en/latest/specifications/binary-distribution-format/#appendix)

Sounds good, I did make note of this option from the responses in the linked stackoverflow discussion, however as python truncates the excess padding I figured it was arbitrary to add the exact right amount of padding, since it can only ever be at most 2, I can make the requested changes though.